### DISCUSSION TITLE
Callout in modal position

### DISCUSSION THREAD

**Discussion by amrita-sakhrani** [twilio-labs/paste#2759] [Q&A] [answered]
Hi, as part of the new language preferences coming into the console, we would like to let users know that a language is still in beta mode. I wanted to ask what would be the best practice of including the callout in a modal, is it acceptable for it to be at the bottom if a beta language is selected? Please advise best practice for this scenario. 

<img width="782" alt="callout_options" src="https://user-images.githubusercontent.com/113033221/197185441-a2637024-97e1-4519-8982-6e80befaeb95.png">

**Comment by gloriliale** [comment]
Hi @amrita-sakhrani! Thank you for the visuals you provided. Could you also post screenshots, or the figma file itself, of the flow this modal is a part of? The context would help us make sure we've thought of a usable solution. Thanks!

**Comment by amrita-sakhrani** [comment, marked as answer]
Hi, @gloriliale thank you for your response. You can view the Figma flow here: https://www.figma.com/file/SlOYUexspe2eQG5nZRjxNn/Locale-Selection?node-id=1435%3A324988. 
To add some context, this modal is an interim solution. The language preference will migrate inside the user settings page, once that page is redesigned and migrated to paste. 
There will be two phases. The first, where the language is in beta mode (and badge shows inside the combobox item). The second, when the language is GA, no badge, but still not yet 100% localized. Thus, a message might remain static at the top of the modal setting the users expectation that there may still be sections in English. https://www.figma.com/file/SlOYUexspe2eQG5nZRjxNn/Locale-Selection?node-id=1552%3A337985 
Thanks and look forward to hearing your thoughts!

**Reply by shleewhite** [reply]
👋🏽 @amrita-sakhrani -

Looking at the Figma, for V3: if beta is important for users to know, although we don't support having a badge in a select option you can include beta in the language name (ex. "Português (Brasil) - Beta").

In general, when you need to add more content to a page, it is best practice to add that content below where the user currently is in the page. This is because it makes it more likely for them to find that information.

**Reply by amrita-sakhrani** [reply]
Thank you for the advice! Will move towards having the callout below the selection option.
